manifests/kustomize/base/installs/generic/postgres/pipeline-install-config.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pipeline-install-config
+data:
+  warning: |
+    1. Do not use kubectl to edit this configmap, because some values are used
+    during kustomize build. Instead, change the configmap and apply the entire
+    kustomize manifests again.
+    2. After updating the configmap, some deployments may need to be restarted
+    until the changes take effect. A quick way to restart all deployments in a
+    namespace: `kubectl rollout restart deployment -n <your-namespace>`.
+  appName: pipeline
+  appVersion: 2.0.0
+  dbType: postgres
+  dbHost: postgres
+  dbPort: "5432"
```

Comment:
Thank you for catching this. I should have added the postgres counterpart for the db config. The original parameters `dbType` and `dbHost` are kept for now because the v2 images these manifests use still call for them. They should be removed after we release the new images.